### ISSUE TITLE
black overlay

### ISSUE THREAD

**Issue by ehsannarmani** [dsa28s/compose-video#3]
is there any way to hide video player back overlay before loading media item? (im using network media item)

**Comment by dsa28s** [comment]
@ehsannarmani  Hello! Thank you for submit issue! What do you mean a back overlay?? Do you mean the black screen??

**Comment by jordan-jakisa** [comment]
Yes, i agree! It would be really helpful to customize the background of the media player. I am facing a similar issue where the black frame sticks out from my video

![image](https://github.com/dsa28s/compose-video/assets/72340216/e8019f9f-df14-4679-b644-4edba58c9a25)
